spec/lib/formatter/csv/v2/annotation_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Formatter::Csv::V2::Annotation do
+  let(:workflow_version) { build_stubbed(:workflow_version, workflow: workflow, tasks: workflow.tasks, strings: workflow.strings) }
+  let(:classification) do
+    build_stubbed(:classification, subjects: [], workflow: workflow, annotations: [annotation])
+  end
+  let(:cache) { instance_double('ClassificationDumpCache') }
+  let(:formatter) do
+    described_class.new(classification, classification.annotations[0], cache)
+  end
+
+  before do
+    maj, min = classification.workflow_version.split('.').map(&:to_i)
+    allow(cache).to receive(:workflow_at_version).with(workflow, maj, min).and_return(workflow_version)
+  end
+
+  context 'with a non-dropdown task workflow and annotation' do
+    let(:workflow) { build_stubbed(:workflow) }
+    let(:annotation) { { 'task' => 'interest', 'value' => {} } }
+    let(:default_formatter) { formatter.default_formatter }
+
+    before do
+      allow(default_formatter).to receive(:to_h)
+    end
+
+    it 'uses the default annotation formatter' do
+      formatter.to_h
+      expect(default_formatter).to have_received(:to_h)
+    end
+  end
+
+  context 'with a v2 dropdown (simple) task' do
+    # details on what allowCreate wf task values for dropdowns do
```

Comment:
Does a user-provided selection, `'option' => false` case deserve its own spec?

Reply:
In theory this is impossible in the new classifier as it stands, https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/plugins/tasks/SimpleDropdownTask/README.md#dev-notes

So the annotation result would always be `'option' => true`

that said - 👍🏼 I'll add a spec for this anyway as the code exists